src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,44 @@
 
 class Animal {
   // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
```

Comment:
think we can declare it not in constructor

Reply:
Maybe yes, but I like to visually separate the methods and properties. I don't know is it a mistake or not, it's just looks more understandable for me

Reply:
don't sure about difference, but Misha in his video use it. probaly you rigth and this have no difference. for me it sounds good practice to get const value from constructor